frontend/demo/src/components/SimpleIconChannel/index.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import {Channel} from 'httpclient';
+import {ReactComponent as GoogleIcon} from '../../assets/images/icons/google_avatar.svg';
+import {ReactComponent as WhatsappIcon} from '../../assets/images/icons/whatsapp_avatar.svg';
+import {ReactComponent as SmsIcon} from '../../assets/images/icons/sms_avatar.svg';
+import {ReactComponent as FacebookIcon} from '../../assets/images/icons/messenger_avatar.svg';
+import {useState} from 'react';
+
+import styles from './index.module.scss';
+
+const FB_GRAPH_API_URL = 'AiryConfig.FB_GRAPH_API_URL';
```

Comment:
I think we should get rid of the Facebook code in this file, what do you think? :)

Reply:
Yeah I agree